scripts/produceData.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# ./produceData.sh $LABEL $PROC_MODIFIER
+
+# $1 label "ref" or "test"
+# $2 procModifier 
+
+echo "label " $1
+echo "procModifiers " $2
+
+source /cvmfs/cms.cern.ch/cmsset_default.sh
+echo $PWD
+eval `scramv1 runtime -sh`
+if [[ -n $2 ]] #if not empty
+then
+  cmsDriver.py hgcal_tpg_validation -n 50 + \
+  --mc --eventcontent FEVTDEBUG --datatier GEN-SIM-DIGI-RAW + \
+  --conditions auto:phase2_realistic + \
+  --beamspot HLLHC14TeV + \
+  --step USER:Validation/HGCalValidation/hgcalRunEmulatorValidationTPG_cff.hgcalTPGRunEmulatorValidation + \
+  --geometry Extended2026D49 --era Phase2C9 --procModifiers $2 + \
+  --inputCommands "keep *","drop l1tEMTFHit2016Extras_simEmtfDigis_CSC_HLT","drop l1tEMTFHit2016Extras_simEmtfDigis_RPC_HLT","drop l1tEMTFHit2016s_simEmtfDigis__HLT","drop l1tEMTFTrack2016Extras_simEmtfDigis__HLT","drop l1tEMTFTrack2016s_simEmtfDigis__HLT" + \
+  --filein file:/data_CMS_upgrade/becheva/data_jenkins/003F2BA9-0D02-5A43-A53F-4161E513BFA6.root + \
```

Comment:
- Would be better to put the data in a "non-user" directory
- Need to keep track of the dataset name. At the moment it is not clear what these data are.

Reply:
Yes, it is better to use "non-user" directory. I will move the data file to /data_CMS_upgrade/data_jenkins.

Reply:
Thanks. Could it be moved in a subdirectory made of the dataset name?

Reply:
It was already done yesterday